js/module_loading_test.go

```diff
@@ -634,6 +637,9 @@ func TestStarImport(t *testing.T) {
 		if (typeof k6.sleep != "function") {
 			throw "k6.sleep has wrong type" + typeof k6.sleep;
 		}
+		if (typeof http.get != "function") {
+			throw "http.get has wrong type" + typeof http.get;
+		}
```

Comment:
Maybe import the default export as well and check that they have the same keys?

That will also help with checking if we do not export too much.

Reply:
Sure, what do you think of this change: f492b4033a07d65ea3d910498a4f7b948d5273c7